src/extend.rs

```diff
@@ -1,11 +1,25 @@
+//! Structured extensions for building and parsing Thing Descriptions
+//!
+//! The Thing Description can be extended adding additional ontologies in its [@context](https://www.w3.org/TR/json-ld11/#the-context).
```

Comment:
_adding additional_ is not the best form, but I am not able to suggest a better one in this case :confused:

Reply:
`extended with` sounds better indeed.